README.md

```diff
@@ -1,9 +1,9 @@
-# RunTimeTerror
```

Comment:
This is a little weird - it seems the contents of the file hasn't changed but Git is picking it up as a complete change? For the README, this is not a big deal, but it will certainly get hairy if this starts happening to code!